### PR TITLE
dumpling: add export-or-backup-using-dumpling.md

### PR DESCRIPTION
**Pull request by yikeke**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [x] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

**If you select two or more versions from above**, to trigger the bot to cherry-pick this PR to your desired release version branch(es), you **must** add corresponding labels such as **needs-cherry-pick-4.0**, **needs-cherry-pick-3.1**, **needs-cherry-pick-3.0**, and **needs-cherry-pick-2.1**.

### What is the related PR or file link(s)?

- This PR is translated from: https://github.com/pingcap/docs-cn/pull/3151/, https://github.com/pingcap/docs-cn/pull/3517
- Other reference link(s):

### CODE REVIEW

**Comment by yikeke** [comment]
PTAL @YuJuncen

File: export-or-backup-using-dumpling.md [hunk 1]
```diff
@@ -0,0 +1,122 @@
+---
+title: Export or Backup Data Using Dumpling
+summary: Use the Dumpling tool to export or backup data in TiDB.
+category: how-to
+---
+
+# Export or Backup Data Using Dumpling
+
+This document introduces how to use the [Dumpling](https://github.com/pingcap/dumpling) tool to export or backup data in TiDB. Dumpling exports data stored in TiDB as SQL or CSV data files and can be used to complete logical full backup or export.
```
Comment (YuJuncen) [inline export-or-backup-using-dumpling.md:9]: "complete" sounds a little strange... Maybe 'make a' here would be better?
```suggestion
This document introduces how to use the [Dumpling](https://github.com/pingcap/dumpling) tool to export or backup data in TiDB. Dumpling exports data stored in TiDB as SQL or CSV data files and can be used to make a logical full backup or export.
```

File: export-or-backup-using-dumpling.md [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+  -H 127.0.0.1 \
+  -o /tmp/test \
+  --where "id < 100"
+```
+
+> **Note:**
+> 
+> Currently, Dumpling does not support exporting only certain tables specified by users (i.e. `-T` flag, see [this issue](https://github.com/pingcap/dumpling/issues/76)). If you do need this feature, you can use [MyDumper](/backup-and-restore-using-mydumper-lightning.md) instead.
+
+The exported file is stored in the `. /export-<current local time>` directory by default. Commonly used parameters are as follows:
```
Comment (YuJuncen) [inline export-or-backup-using-dumpling.md:71]: Seems here shouldn't be a space, does it presents at Chinese version of document too? If does, sorry about it.

```suggestion
The exported file is stored in the `./export-<current local time>` directory by default. Commonly used parameters are as follows:
```

Comment (yikeke) [reply export-or-backup-using-dumpling.md]: No, it's my fault. Updated.

**Comment by yikeke** [comment]
PTAL @lilin90

**Pull request merged**